docs/src/main/paradox/spring-web.md

```diff
@@ -0,0 +1,39 @@
+# Spring Web
+
+Spring 5.0 introduced compatibility with the [Reactive Streams](https://github.com/reactive-streams/reactive-streams-jvm) 
```

Comment:
Maybe link to http://www.reactive-streams.org/ for this audience?

Reply:
yeah, good point